public/app/plugins/panel/geomap/style/utils.ts

```diff
@@ -1,18 +1,64 @@
-import { StyleConfig } from './types';
+import { config } from '@grafana/runtime';
+import { TextDimensionMode } from 'app/features/dimensions';
+import { getMarkerMaker } from './markers';
+import { defaultStyleConfig, StyleConfig, StyleConfigFields, StyleConfigState } from './types';
+
+/** Indicate if the style wants to show text values */
+export function styleUsesText(config: StyleConfig): boolean {
```

Comment:
@nmarrs -- we should try to replace our various similar checks with this call.  (and perhaps improve the function name!) :